routers/web/repo/branch.go

```diff
@@ -68,15 +52,16 @@ func Branches(ctx *context.Context) {
 	}
 	pageSize := setting.Git.BranchesRangeSize
 
-	skip := (page - 1) * pageSize
-	log.Debug("Branches: skip: %d limit: %d", skip, pageSize)
-	defaultBranchBranch, branches, branchesCount := loadBranches(ctx, skip, pageSize)
-	if ctx.Written() {
+	log.Debug("Branches: page: %d pageSize: %d", page, pageSize)
```

Comment:
Although it's a debug log, why it is needed?

Reply:
derived from line 72.